libs/service-portal/applications/src/index.ts

```diff
@@ -15,7 +15,7 @@ export const applicationsModule: ServicePortalModule = {
     {
       name: m.heading,
       path: ServicePortalPath.ApplicationRoot,
-      enabled: userInfo.scopes.includes(ApplicationScope.read),
+      enabled: userInfo?.scopes?.includes(ApplicationScope.read),
```

Comment:
the type signature of the `routes` method states that `userInfo` is required.

If it isn't always present in the props, then there's a bug elsewhere and re responsible thing to do is to trace the origin of the bug and fix it.

Reply:
The bug may very likely be caused by a `strict: false` setting in some up-stream module's `tsconfig`.

Look out for that.

Reply:
...or there might be a nasty type assertion somewhere forcing an incorrectly shaped/half-formed object into a tsc state that then gets an undeserved trust thereafter.

Reply:
The userInfo is actually there, it's just that the scopes are not available for a mock user.
I'll just change that and undo this change.